app/services/disability_claim_service.rb

```diff
@@ -90,7 +90,7 @@ def claims_scope
 
   def create_or_update_claim(raw_claim)
     claim = claims_scope.where(evss_id: raw_claim['id']).first_or_initialize(data: {})
-    claim.update_attributes(data: claim.data.merge(raw_claim), successful_sync: true)
```

Comment:
I think I had this backwards in our code now, but the way I was thinking about it was that the details call would always replace the data we have for the claim as it has the most data. The list call should only update the fields it contains if the user goes back and new data is fetched. Looking at it now, there are some nested hashes that are probably causing issues when going back to the list view and wiping out the attribute from the details. The end goal is that after making the list and details calls, we should always be able to reconstruct both views from the data in the DB. In your first scenario, I think that should cause an error, if the claim only contains data from the EVSS list call, there is not enough info for the details page. Quick solution may be storing 2 JSON blobs and throwing an error on the details page when EVSS is down and we don't have the details blob.